### PR TITLE
Change SEP-18 + SEP-29 to Active

### PR DESCRIPTION
**Pull request by rice2000**
- Change SEP status on each doc
- Update README to move each from draft table to active table

### CODE REVIEW

**Comment by rice2000** [comment]
@ire-and-curses or @leighmcculloch: if this looks good, can one of you merge it?  I don't have write access.

File: ecosystem/README.md [hunk 1]
```diff
@@ -53,15 +55,14 @@
 | [SEP-0015](sep-0015.md) | Attachment Convention | Interstellar | Standard | Draft |
 | [SEP-0016](sep-0016.md) | Account Transfer Permissionless Payment Protocol (@p2p) | Jeremy Rubin | Standard | Draft |
 | [SEP-0017](sep-0017.md) | Issuer account funding protocol (CAP-13 Based) | Tom Quisel | Standard | Draft |
-| [SEP-0018](sep-0018.md) | Data Entry Namespaces | Mister.Ticot | Standard | Draft |
```
Comment (chalermchai29) [inline ecosystem/README.md:56]: | [SEP-0016](sep-0016.md) | Account Transfer Permissionless Payment Protocol (@p2p) | Jeremy Rubin | Standard | Draft |

Write Preview
 
Attach files by dragging & dropping, selecting or pasting them.
Start a review
Add single comment
Cancel


| [SEP-0017](sep-0017.md) | Issuer account funding protocol (CAP-13 Based) | Tom Quisel | Standard | Draft |


Write Preview
 
Attach files by dragging & dropping, selecting or pasting them.
Start a review
Add single comment
Cancel
Write Preview

Nothing to preview

Start a review
Add single comment
Cancel


| [SEP-0019](sep-0019.md) | Bootstrapping Multisig Transaction Submission | Paul Selden, Nikhil Saraf | Standard | Draft |

| [SEP-0021](sep-0021.md) | On-chain signature & transaction sharing | Mister.Ticot | Informational | Draft |

| [SEP-0022](sep-0022.md) | IPFS Support | Samuel B. Sendelbach | Informational | Draft |

| [SEP-0023](sep-0023.md) | Augmented strkey format for multiplexed addresses | David Mazières and Tomer Weller | Standard | Draft |


| [SEP-0030](sep-0030.md) | Recoverysigner: multi-party key management of Stellar accounts | Leigh McCulloch, Lindsay Lin | Standard | Draft |










### Rejected and Deprecated Proposals




| Number | Title | Author | Track | Status |

**Review by chalermchai29 — COMMENTED** [review]
| [SEP-0016](sep-0016.md) | Account Transfer Permissionless Payment Protocol (@p2p) | Jeremy Rubin | Standard | Draft |

Write Preview
 
Attach files by dragging & dropping, selecting or pasting them.
Start a review
Add single comment
Cancel


| [SEP-0017](sep-0017.md) | Issuer account funding protocol (CAP-13 Based) | Tom Quisel | Standard | Draft |


Write Preview
 
Attach files by dragging & dropping, selecting or pasting them.
Start a review
Add single comment
Cancel
Write Preview

Nothing to preview

Start a review
Add single comment
Cancel


| [SEP-0019](sep-0019.md) | Bootstrapping Multisig Transaction Submission | Paul Selden, Nikhil Saraf | Standard | Draft |

| [SEP-0021](sep-0021.md) | On-chain signature & transaction sharing | Mister.Ticot | Informational | Draft |

| [SEP-0022](sep-0022.md) | IPFS Support | Samuel B. Sendelbach | Informational | Draft |

| [SEP-0023](sep-0023.md) | Augmented strkey format for multiplexed addresses | David Mazières and Tomer Weller | Standard | Draft |


| [SEP-0030](sep-0030.md) | Recoverysigner: multi-party key management of Stellar accounts | Leigh McCulloch, Lindsay Lin | Standard | Draft |










### Rejected and Deprecated Proposals




| Number | Title | Author | Track | Status |

File: ecosystem/README.md [hunk 2]
```diff
@@ -53,15 +55,14 @@
...
 | [SEP-0019](sep-0019.md) | Bootstrapping Multisig Transaction Submission | Paul Selden, Nikhil Saraf | Standard | Draft |
 | [SEP-0021](sep-0021.md) | On-chain signature & transaction sharing | Mister.Ticot | Informational | Draft |
 | [SEP-0022](sep-0022.md) | IPFS Support | Samuel B. Sendelbach | Informational | Draft |
 | [SEP-0023](sep-0023.md) | Augmented strkey format for multiplexed addresses | David Mazières and Tomer Weller | Standard | Draft |
-| [SEP-0029](sep-0029.md) | Account Memo Requirements | OrbitLens, Tomer Weller, Leigh McCulloch, David Mazières | Standard | Draft |
```
Comment (chalermchai29) [inline ecosystem/README.md:61]: | [SEP-0023](sep-0023.md) | Augmented strkey format for multiplexed addresses | David Mazières and Tomer Weller | Standard | Draft |

**Review by chalermchai29 — COMMENTED** [review]
| [SEP-0016](sep-0016.md) | Account Transfer Permissionless Payment Protocol (@p2p) | Jeremy Rubin | Standard | Draft |

Write Preview
 
Attach files by dragging & dropping, selecting or pasting them.
Start a review
Add single comment
Cancel


| [SEP-0017](sep-0017.md) | Issuer account funding protocol (CAP-13 Based) | Tom Quisel | Standard | Draft |


Write Preview
 
Attach files by dragging & dropping, selecting or pasting them.
Start a review
Add single comment
Cancel
Write Preview

Nothing to preview

Start a review
Add single comment
Cancel


| [SEP-0019](sep-0019.md) | Bootstrapping Multisig Transaction Submission | Paul Selden, Nikhil Saraf | Standard | Draft |

| [SEP-0021](sep-0021.md) | On-chain signature & transaction sharing | Mister.Ticot | Informational | Draft |

| [SEP-0022](sep-0022.md) | IPFS Support | Samuel B. Sendelbach | Informational | Draft |

| [SEP-0023](sep-0023.md) | Augmented strkey format for multiplexed addresses | David Mazières and Tomer Weller | Standard | Draft |


| [SEP-0030](sep-0030.md) | Recoverysigner: multi-party key management of Stellar accounts | Leigh McCulloch, Lindsay Lin | Standard | Draft |










### Rejected and Deprecated Proposals




| Number | Title | Author | Track | Status |

**Pull request merged**